src/test/java/com/hocs/test/pages/create_case/SuccessfulCaseCreation.java

```diff
@@ -44,11 +51,20 @@ public void assertCaseCreatedSuccess() {
 
     // Set Case Reference Number to session variable during case creation
 
+//    public String getCaseReference() {
+//        String wholeString = createdCaseMessage.getText();
+//        String to_remove="Created a new case: ";
+//
+//        String caseReference = wholeString.replace(to_remove, "");
+//        setSessionVariable("caseReference").to(caseReference);
+//
+//        return caseReference;
+//    }
+
     public String getCaseReference() {
-        String wholeString = createdCaseMessage.getText();
-        String to_remove="Created a new case: ";
+        String wholeString = workstacks.caseReferenceOnAllocationScreen.getText();
 
-        String caseReference = wholeString.replace(to_remove, "");
+        String caseReference = wholeString;
         setSessionVariable("caseReference").to(caseReference);
```

Comment:
I think you can remove the wholeString line entirely so this can now just be

`public String getCaseReference() {
String caseReference = workstacks.caseReferenceOnAllocationScreen.getText();
return caseReference;
}
```